### PR TITLE
Created Test Scene

### PR DESCRIPTION
**Pull request by MattsAttack**
Made some good looking water
added a platform

Still trying to figure out godot 3d lol. This template is kind of confusing but I'm getting there

- Windows 10

### CODE REVIEW

File: addons/godot-xr-tools/objects/viewport_2d_in_3d.gd [hunk 1]
```diff
@@ -199,7 +199,7 @@ func _on_pointer_event(event : XRToolsPointerEvent) -> void:
 
 # Handler for input eventsd
 func _input(event):
-	$Viewport.push_input(event)
+	$Viewport.push_inRput(event)
```
Comment (lishaduck) [inline addons/godot-xr-tools/objects/viewport_2d_in_3d.gd:202]: Is this a typo?

File: assets/water_stuff/WaterPlane.gdshader
Comment (lishaduck) [inline assets/water_stuff/WaterPlane.gdshader:1]: Ewww. Well, shaders it is, I guess. Is this a blender import? I think we're trying to use Blender to keep the models "agnostic."

File: assets/water_stuff/shader_complex.tres [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+shader_parameter/noise_scale = 10.0
+shader_parameter/height_scale = 0.15
+shader_parameter/color_deep = null
+shader_parameter/color_shallow = null
+shader_parameter/beers_law = 2.0
+shader_parameter/depth_offset = -0.75
+shader_parameter/edge_scale = 0.1
+shader_parameter/near = 1.0
+shader_parameter/far = 100.0
+shader_parameter/edge_color = null
```
Comment (lishaduck) [inline assets/water_stuff/shader_complex.tres:50]: It's nice that that's all built in.

File: project.godot [hunk 3]
```diff
@@ -66,7 +66,6 @@ common/drop_mouse_on_gui_input_disabled=true
 
 [rendering]
 
-renderer/rendering_method="gl_compatibility"
```
Comment (lishaduck) [inline project.godot:69]: Good catch!

File: game/zones/outside/outside_zone.tscn [hunk 4]
```diff
@@ -121,59 +121,17 @@ size = Vector3(1, 3, 0.2)
...
-bones/23/rotation = Quaternion(0.00687177, -0.00357275, -0.211953, 0.977249)
-
 [node name="BoneAttachment3D" parent="XROrigin3D/LeftHand/CollisionHandLeft/LeftHand/Hand_Nails_low_L/Armature/Skeleton3D" index="1"]
-transform = Transform3D(0.54083, 0.840813, -0.0231736, -0.0826267, 0.0805243, 0.993322, 0.837064, -0.535303, 0.113023, 0.039902, 0.0402828, -0.150096)
+transform = Transform3D(0.54083, 0.840812, -0.0231736, -0.0826267, 0.0805244, 0.993322, 0.837063, -0.535304, 0.113024, 0.0399019, 0.0402829, -0.150096)
 
 [node name="AnimationTree" parent="XROrigin3D/LeftHand/CollisionHandLeft/LeftHand" index="1"]
-tree_root = SubResource("AnimationNodeBlendTree_qrs50")
-
-[node name="Skeleton3D" parent="XROrigin3D/RightHand/CollisionHandRight/RightHand/Hand_Nails_low_R/Armature" index="0"]
```
Comment (lishaduck) [inline game/zones/outside/outside_zone.tscn:151]: Just curious, what did this do before?

**Review by lishaduck — COMMENTED** [review]
Just some questions.

**Review by lishaduck — DISMISSED** [review]

File: game/zones/outside/outside_zone.tscn (hunk 4 above)
Comment (MattsAttack) [reply game/zones/outside/outside_zone.tscn:151]: I didn't change this. Also we're not using the provided scenes so it doesn't impact us

File: assets/water_stuff/WaterPlane.gdshader
Comment (MattsAttack) [reply assets/water_stuff/WaterPlane.gdshader:1]: Nope made it in godot

File: game/zones/outside/outside_zone.tscn (hunk 4 above)
Comment (lishaduck) [reply game/zones/outside/outside_zone.tscn:151]: I know. I'm just curious what this did. If you don't know, I can just leave it at that.

**Comment by lishaduck** [comment]
Can you show a screenshot of the sky?

**Comment by lishaduck** [comment]
Tried this, and it crashes. We can discuss it in class.

**Comment by lishaduck** [comment]
Ok, just about finished.

**Comment by lishaduck** [comment]
FYI, this is 99% merged. The sky is still failing compilation, but I'm going to work on that as soon as I finish some notes for history.

**Pull request closed without merging**